src/Mvc/test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/MemberExpressionCacheKeyTest.cs

```diff
@@ -1,111 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq.Expressions;
-using System.Reflection;
-using Xunit;
-
-namespace Microsoft.AspNetCore.Mvc.ViewFeatures
-{
-    public class MemberExpressionCacheKeyTest
```

Comment:
Unintentional deletion of test coverage?

Reply:
Good catch. Adding it back 👍